oak-store-document/pom.xml

```
@@ -43,8 +43,8 @@
         <configuration>
           <instructions>
             <Import-Package>
-              com.mongodb*;version="[3.8, 4)";resolution:=optional,
-              org.bson*;version="[3.8, 4)";resolution:=optional,
+              com.mongodb*;version="[5.0, 5.2)";resolution:=optional,
```

Since we are switching to `5.2.0`, shouldn't it be

```suggestion
              com.mongodb*;version="[5.2, 6)";resolution:=optional,
```